Makefile

```diff
@@ -4,9 +4,12 @@ SERVER=""
 install:
 	echo -e "import os\ntry:\n  if not os.listdir('/usr/src'): exit('/usr/src must be populated!')\nexcept FileNotFoundError:\n  exit('/usr/src must be populated!')" | python3.6
 	git pull
+	git submodule init
+	git submodule update
```

Comment:
These two can be combined into `git pull --recursive`, so even though we're ditching GitPython as a submodule, I did forget to do this in the makefile :D So we should modify this change and include it in the PR still.